plexapi/library.py

```diff
@@ -1093,20 +1116,21 @@ class Collections(PlexObject):
     def _loadData(self, data):
         self.ratingKey = utils.cast(int, data.attrib.get('ratingKey'))
         self._details_key = "/library/metadata/%s%s" % (self.ratingKey, self._include)
+        self.addedAt = utils.toDatetime(data.attrib.get('addedAt'))
+        self.childCount = utils.cast(int, data.attrib.get('childCount'))
+        self.collectionMode = data.attrib.get('collectionMode')
+        self.collectionSort = data.attrib.get('collectionSort')
+        self.fields = self.findItems(data, etag='Field')
         self.key = data.attrib.get('key')
-        self.type = data.attrib.get('type')
-        self.title = data.attrib.get('title')
+        self.index = utils.cast(int, data.attrib.get('index'))
+        self.maxYear = utils.cast(int, data.attrib.get('maxYear'))
+        self.minYear = utils.cast(int, data.attrib.get('minYear'))
         self.subtype = data.attrib.get('subtype')
         self.summary = data.attrib.get('summary')
-        self.index = utils.cast(int, data.attrib.get('index'))
         self.thumb = data.attrib.get('thumb')
-        self.addedAt = utils.toDatetime(data.attrib.get('addedAt'))
+        self.title = data.attrib.get('title')
+        self.type = data.attrib.get('type')
         self.updatedAt = utils.toDatetime(data.attrib.get('updatedAt'))
-        self.childCount = utils.cast(int, data.attrib.get('childCount'))
-        self.minYear = utils.cast(int, data.attrib.get('minYear'))
-        self.maxYear = utils.cast(int, data.attrib.get('maxYear'))
-        self.collectionMode = data.attrib.get('collectionMode')
-        self.collectionSort = data.attrib.get('collectionSort')
 
```

Comment:
Maybe for a different PR:

Add attributes `guid`, `titleSort`, `librarySectionTitle`, `librarySectionID`, `contentRating`, `summary`.

Reply:
added